spikes/kafka-flink/README.rst

```diff
@@ -64,6 +64,40 @@ to them by using:
 :Apache Flink Dashboard: http://localhost:8081/
 :CrateDB Admin UI: http://localhost:4200/
 
+.. note::
+
+    CrateDB uses a mmapfs directory by default to store its indices. The
+    default operating system limits on mmap counts is likely to be too low,
+    which may result in out of memory exceptions.
+
+    On Linux, you can increase the limits by running the following command::
+
+        sudo sysctl -w vm.max_map_count=262144
+
+    To set this value permanently, update the ``vm.max_map_count`` setting in
+    ``/etc/sysctl.conf``. To verify after rebooting, run
+    ``sysctl vm.max_map_count``.
+
+
+.. note::
```

Comment:
I think Naomi suggested to use *Note* in one of last PR-s but I don't remember which one

Reply:
Hm. I don't know exactly what you mean here. I believe `.. note::` is a well-known directive in Sphinx to create an emphasized block of text?

Reply:
I found the [comment](https://github.com/crate/crate/pull/11288#discussion_r617444464) but I'm not 100% it's applicable here just recalled something similar and thought it could be related.

Reply:
Ah right, thank you. This is correct, GitHub RST rendering doesn't support admonitions (e.g., NOTE). I didn't care about this too much, but now, when looking at the results [1], I believe you are right that we should improve the situation for the benefit of all people actually reading this on GitHub.

So, thanks for being persistent on this matter. I just added e5650664.

[1] https://github.com/crate/cratedb-examples/blob/d2a8043c0d959d73c714f97057d7db7493e755e7/spikes/kafka-flink/README.rst#foundation-infrastructure